### DISCUSSION TITLE
HTMLの初心者向けディスカッション

### DISCUSSION THREAD

**Discussion by Shinshin100** [Shinshin100/Introduction-to-HTML#1] [Announcements]
HTML初心者向けのコミュニティです。
* 疑問に思っていることは何でも質問してください。
* アイデアを共有してください。
* 他のコミュニティメンバーと交流してください。
* 他の人を歓迎し、オープンな気持ちで接してください。
ここは私たちが一緒に構築するコミュニティです。

**Comment by shino1486** [comment]
疑問：そもそもHFMLってなんなのか？？PC言語の方式？

交流：しんちゃーん！　これでりぼじとりー付くようになったらADSノーベル賞！（笑）

**Reply by Shinshin100** [reply]
コメントありがとうございます。
HTMLとは、Webサイトを作成するために使われる言語です。

**Comment by yama15** [comment]
お疲れ様です　yamaです

**Comment by koumoto112** [comment]
こんばんは　みら吉です
HTML　ググったけどよく分かりません(笑)

**Comment by ads-light1992** [comment]
こんばんは、lightです
色々調べて頂いてありがとうございます！

**Comment by hiro-ito1** [comment]
dflatです！
いろいろありがとうございます！

**Comment by mashi198** [comment]
MASAです。
書き込んでみました

**Comment by mashi198** [comment]
MASAです
１分前だと早すぎるのかもしれませんがコントリビューション増えませんでした

**Comment by Shinshin100** [comment]
お疲れ様です。
不明点がありましたら、ご連絡ください。
よろしくお願いします。
